docs/designers-developers/developers/tutorials/metabox/meta-block-2-register-meta.md

```diff
@@ -0,0 +1,19 @@
+# Register Meta Field
+
+A post meta field is a WordPress object used to store extra data about a post. You need to first register a new meta field prior to use.
+
+When registering the field, note the `show_in_rest` parameter, this ensures the data will be included in the REST API, which the Block Editor uses to load and save meta data. See the [register_meta](https://developer.wordpress.org/reference/functions/register_meta/) function definition for extra information.
+
+To register the field, add the following PHP code to your plugin file:
+
+```php
+// register custom meta tag field
+function myguten_register_meta() {
+	register_meta( 'post', 'myguten_meta_block_field', array(
```

Comment:
Why not `register_post_meta`?

Reply:
I find `register_post_meta` to be slightly more complicated since the syntax requires passing an empty string in for the post type to cover all types. 

Whereas using `register_meta()` and passing in `post` covers all post types by default.

I don't feel that strongly about it, so could switch easily.